graphql/executor/executor.go

```diff
@@ -24,7 +25,8 @@ type Executor struct {
 	recoverFunc    graphql.RecoverFunc
 	queryCache     graphql.Cache[*ast.QueryDocument]
 
-	parserTokenLimit int
+	parserTokenLimit  int
+	disableSuggestion bool
```

Comment:
Unless `SetDisableSuggestion(true)` is called, the behavior will remain exactly the same as before, so it does not affect backward compatibility. Only those who are aware of this feature will be able to turn off the suggestions.